src/text-transform/transformers.ts

```diff
@@ -17,7 +17,7 @@ export type ContentTransformerProps = {
         blocks?: ConstructorBlock[];
     };
     options: {
-        lang: Lang;
+        lang: `${Lang}`;
```

Comment:
could we define this lang type once in models like `type Lang = `${Lang}`?

Reply:
I think this is not necessary because we will add another level of abstraction and we will have to go to the place of the announcement to understand what exactly Lang is used, dev experience will deteriorate